build/ci/pr/Jenkinsfile

```diff
@@ -72,24 +72,21 @@ pipeline {
                         }
                     }
                     steps {
-                        createConfig()
-                        sh """
-                            cat >deployer-config.yml <<EOF
-id: gke-ci
-overrides:
-  kubernetesVersion: "1.12"
-  clusterName: eck-pr-$BUILD_NUMBER
-  vaultInfo:
-    address: $VAULT_ADDR
-    roleId: $VAULT_ROLE_ID
-    secretId: $VAULT_SECRET_ID
-  gke:
-    gCloudProject: $GCLOUD_PROJECT
-EOF
-                            echo "Setting CRD_FLAVOR to trivial-versions to support K8s < 1.14"
-                            echo "CRD_FLAVOR = trivial-versions" >> .env
-                            make -C build/ci TARGET=ci-e2e ci
-                        """
+                        script {
+                            createConfig()
+                            createDeployerConfig()
+                            sh """
+                                echo "Setting CRD_FLAVOR to trivial-versions to support K8s < 1.14"
```

Comment:
1.13 doesnt require trivial versions, it just doesnt do anything with the extra version info it receives

Reply:
Do you mean that this block can be removed? It's already in PR job https://github.com/elastic/cloud-on-k8s/blob/master/build/ci/pr/Jenkinsfile#L89 and I just keep it. I can remove it if it's not needed

Reply:
Fixed as part of Thibault comments